website/docs/r/function_app.html.markdown

```diff
@@ -146,6 +146,8 @@ The following arguments are supported:
 
 * `client_affinity_enabled` - (Optional) Should the Function App send session affinity cookies, which route client requests in the same session to the same instance?
 
+* `client_cert_mode` - (Optional) The mode of the Function App's client certificates requirement for incoming requests. Possible values are "Required", "Optional", and "Ignore". Defaults to "Ignore".
```

Comment:
since this becomes an Optional field, we can make this:

```suggestion
* `client_cert_mode` - (Optional) The mode of the Function App's client certificates requirement for incoming requests. Possible values are `Required` and `Optional`.
```

(we also use backticks rather than quotation marks to quote field names, like this)